opentelekomcloud/acceptance/cce/resource_opentelekomcloud_cce_addon_v3_test.go

```diff
@@ -63,18 +63,19 @@ func TestAccCCEAddonV3ForceNewCCE(t *testing.T) {
 	})
 }
 
-func TestAccCCEAddonV3EmptyBasic(t *testing.T) {
-	resource.Test(t, resource.TestCase{
-		PreCheck:     func() { common.TestAccPreCheck(t) },
-		Providers:    common.TestAccProviders,
-		CheckDestroy: testAccCheckCCEAddonV3Destroy,
-		Steps: []resource.TestStep{
-			{
-				Config: testAccCCEAddonV3EmptyBasic,
-			},
-		},
-	})
-}
+// // This test is broken due to OTC CCE updates
+// func TestAccCCEAddonV3EmptyBasic(t *testing.T) {
+// 	resource.Test(t, resource.TestCase{
+// 		PreCheck:     func() { common.TestAccPreCheck(t) },
+// 		Providers:    common.TestAccProviders,
+// 		CheckDestroy: testAccCheckCCEAddonV3Destroy,
+// 		Steps: []resource.TestStep{
+// 			{
+// 				Config: testAccCCEAddonV3EmptyBasic,
+// 			},
+// 		},
+// 	})
+// }
```

Comment:
Please remove code if not used

Reply:
I thought that issue could be fixed from the OTC side and we will enable it.